src/structures/GuildScheduledEvent.js

```diff
@@ -216,6 +217,17 @@ class GuildScheduledEvent extends Base {
     return Endpoints.scheduledEvent(this.client.options.http.scheduledEvent, this.guildId, this.id);
   }
 
+  /**
+   * Creates an invite url to this guild scheduled event.
+   * @param {InviteResolvable} invite The invite to the guild channel of this guild scheduled event
+   * <info>For an `EXTERNAL` guild scheduled event, this can be an invite to any guild channel</info>
+   * @returns {string}
+   */
+  createInviteUrl(invite) {
+    const code = DataResolver.resolveInviteCode(invite);
```

Comment:
This should be generating a new invite and take inviteoptions as an argument imo, or maybe have an option to try to get from cache and only create if no invite to the channel is found